app/javascript/components/my-studies/MyStudiesPage.jsx

```diff
@@ -0,0 +1,276 @@
+
+import React, { useEffect, useState } from 'react'
+import { useTable, usePagination, useSortBy, useFilters } from 'react-table'
+import Modal from 'react-bootstrap/lib/Modal'
+
+import UserProvider from '~/providers/UserProvider'
+import LoadingSpinner from '~/lib/LoadingSpinner'
+import { fetchEditableStudies } from '~/lib/scp-api'
+import PagingControl from '~/components/search/results/PagingControl'
+import ErrorBoundary from '~/lib/ErrorBoundary'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faSort, faSortDown, faSortUp, faEllipsisV, faTrash, faExclamationTriangle
+} from '@fortawesome/free-solid-svg-icons'
+import InfoPopup from '~/lib/InfoPopup'
+
+
+// define these outside the render loop so they don't cause rerender loops
+// if they ever need to be dynamic, make sure to use useMemo
+const columns = [{
+  Header: 'Title ',
+  accessor: 'name',
+  Cell: ({ value, row: { original: { accession, description } } }) => {
+    let shortDesc = description.substring(0, 100)
+    if (description.length > 100) {
+      shortDesc += '...'
+    }
+    return <div>
+      <a href={`/single_cell/study/${accession}`}>{value}</a><br/>
+      <span className="detail">{shortDesc}</span>
+
+    </div>
+  },
+  sortType: (rowA, rowB) => {
+    return rowA.original.name.localeCompare(rowB.original.name, 'en', { numeric: true, ignorePunctuation: true })
+  },
+  filter: (rows, id, filterValue) => {
+    const lowerCaseFilter = filterValue.toLowerCase()
+    return rows.filter(row => (
+      row.original.name.toLowerCase().includes(lowerCaseFilter) ||
+      row.original.description.toLowerCase().includes(lowerCaseFilter) ||
+      row.original.ownerEmail.toLowerCase().includes(lowerCaseFilter)
+    ))
+  },
+  Filter: ({ column: { filterValue, setFilter } }) => {
+    return <>
+      &nbsp; &nbsp;
+      <input value={filterValue || ''} style={{ minWidth: '350px', border: 'none', margin: '4px' }}
+        onChange={e => setFilter(e.target.value || undefined)}
+        placeholder="Filter by name, description, or owner email"
+      />
+    </>
+  }
+}, {
+  Header: 'Owner ',
+  accessor: 'ownerEmail',
+  sortType: (rowA, rowB) => {
+    return rowA.original.name.localeCompare(rowB.original.name, 'en', { numeric: true, ignorePunctuation: true })
+  },
+  disableFilters: true
+}, {
+  Header: 'Created ',
+  accessor: 'createdAt',
+  Cell: ({ value }) => {
+    const date = new Date(value)
+    return `${date.getFullYear()}-${date.getMonth()}-${date.getDate()}`
```

Comment:
Blocker: For dates, let's use Terra’s style — <3-letter month> <unpadded day>, <year> — or ISO-8601.  The screenshot’s unpadded ISO-8601 style is unfamiliar to both technical folks (ISO-8601 is padded, e.g. 2022-05-18 instead of 2022-5-18) and non-technical folks.

```suggestion
    return `${date.toISOString().split('T')[0]}`
```

Reply:
done.  Also, I upgraded the text ellipsis styling/cutoff per your comments from demo